### DISCUSSION TITLE
[nuxt] [short-vmodel] [QUESTION] Can't find short-vmodel in Nuxt configuration macros options

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vue-macros/vue-macros#450] [Q&A] [answered]
![image](https://github.com/vue-macros/vue-macros/assets/10191085/d5a086ad-14be-4ed4-b315-3e0331d5e61a)

Of course, as always, I try to debug myself, installed `pnpm add -D @vue-macros/short-vmodel`, and added:

`nuxt.config.ts`
```ts
import vue from '@vitejs/plugin-vue'
import { transformShortVmodel } from '@vue-macros/short-vmodel'

export default defineNuxtConfig({
  vite: {
    plugins: [
    vue({
      template: {
        compilerOptions: {
          nodeTransforms: [
            transformShortVmodel({
              prefix: '$',
            }),
          ],
        },
      },
    }),
  ]
  }
})
```

And got errors.

### Environment
Node: 18.16.1
@vue-macros/nuxt: 1.6.0
nuxt: 3.6.5

**Comment by sxzz** [comment, marked as answer]
Fixed via https://github.com/vue-macros/vue-macros/commit/322321bf76f0d1b38f8fb2c19b6308eb0a3091d4

**Reply by sxzz** [reply]
BTW, you can new an issue to feedback bugs for higher priority.

**Comment by angelhdzmultimedia** [comment]
![image](https://github.com/vue-macros/vue-macros/assets/10191085/e22ae5f5-79db-4f14-98b2-34b35e1189bd)
![image](https://github.com/vue-macros/vue-macros/assets/10191085/ae0ba1b3-9f91-4f07-8264-9e028418db5e)
![image](https://github.com/vue-macros/vue-macros/assets/10191085/fea77982-905f-4230-a4aa-ad123797f6c3)
![image](https://github.com/vue-macros/vue-macros/assets/10191085/147ac9f0-2a88-4cc7-b6ad-98b4060209fc)


### Question
I read somewhere (can't remember where), that short-vmodel works in JSX thanks to someone's contribution.
Am I losing my mind? LOL.

Thanks for the lighting fast work on the setupSFC in layouts and short-vmodel visibility in nuxt config macro options.

**Reply by sxzz** [reply]
short-vmodel doesn't support JSX yet. Maybe will support it in the future.

**Reply by angelhdzmultimedia** [reply]
Noted! Thanks for everything! Wishing you health, peace, and success.
